actions.go

```diff
@@ -56,8 +56,9 @@ const (
 	endGroupCmd = "endgroup"
 
 	debugCmd   = "debug"
-	errorCmd   = "error"
+	noticeCmd  = "notice"
 	warningCmd = "warning"
+	errorCmd   = "error"
```

Comment:
These were sorted alphabetically intentionally. Please sort.

Reply:
Now they are sorted by level, byt the same order as in code below and docs:
![image](https://user-images.githubusercontent.com/11512/147683406-c23c7b3e-4ce5-4ca2-8706-0915c4c4d261.png)